templates/news/news.html

```diff
@@ -16,15 +16,15 @@ <h3>News
         </div>
 
             {% for news in news_objects %}
-            <div class="col-6 my-4">
```

Comment:
This will cause the news item to be full width on all sizes. Perhaps we should use <div class="col-12 col-md-4"> to make them smaller on larger screens?

Reply:
Indeed, I thought it makes sense to keep the news items full width of the page because if we have more formatting there now with markdown and HTML but in two columns it will look like things are all over the place. So I thought news items should then just be one under another.